misc/pytorch_toolkit/breast_ultrasound_simulation/src/infer.py

```diff
@@ -0,0 +1,36 @@
+import os
+from torch.utils import data
+from dataloader import Kaggle_BUS_Dataset
+from solve import solver_inter2d
+from utils.read_config import load_config
+
+
+net_config_path = "configs/net_config.json"
```

Comment:
Please, transform this to a callable script (with `if __name__ == '__main__':` and arguments parsing)